romancal/source_catalog/source_catalog_step.py

```diff
@@ -43,6 +43,7 @@ class SourceCatalogStep(RomanStep):
         ci2_star_threshold = float(default=1.8)  # CI 2 star threshold
         suffix = string(default='cat')        # Default suffix for output files
         fit_psf = boolean(default=True)      # fit source PSFs for accurate astrometry
+        return_updated_model = boolean(default=False)   # should the updated model be returned?
```

Comment:
Can we expand the comment here?  Maybe: should image model be updated and returned, instead of returning the catalog?  I realize one doesn't have much space, though.